graphstorm-processing/pyproject.toml

```diff
@@ -19,6 +19,8 @@ pandas = "^1.3.5"
 psutil = "^5.9.5"
 sagemaker = "^2.83.0"
 scipy = "^1.10.1"
+transformers = "^4.37.1"
+setuptools = "59.6.0"
```

Comment:
This would only be a requirement for EMR-S 7.0.0, which we're not using currently, let's remove it.

Reply:
When we release an EMR 7 Dockerfile we'll handle this particular dependency using a virtualenv on the image instead of modifying the project's dependencies.